keps/sig-scheduling/1923-prefer-nominated-node/kep.yaml

```diff
@@ -15,7 +15,7 @@ approvers:
   - "@Huang-Wei"
   - "@ahg-g"
 stage: alpha
-latest-milestone: "v1.21"
+# latest-milestone: "v1.21"
```

Comment:
@chendave FYI, PRR approval will need to be part of the PR that sets this back to 1.21.

Reply:
Done, thanks for the notification.